UPGRADING.md

```diff
@@ -489,6 +493,39 @@ await ddbDocClient.send(
 );
 ```
 
+### Undefined Values in Marshalling
+
+- In v2 `undefined` values in objects were automatically omitted during the marshalling process to DynamoDB.
+
+- In v3, using [`@aws-sdk/lib-dynamodb`](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/modules/_aws_sdk_lib_dynamodb.html), the default behavior has changed: `undefined` values in objects are now included during marshalling. Developers need to explicitly manage undefined values to exclude them using the `removeUndefinedValues` flag.
+
+Example:
+
+```js
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, PutCommand } from "@aws-sdk/lib-dynamodb";
+import { marshall } from "@aws-sdk/util-dynamodb";
+
+const client = new DynamoDBClient({});
+
+const ddbDocClient = DynamoDBDocumentClient.from(client);
```

Comment:
marshall options should be the 2nd argument to the DocClient factory, not separately given to the marshall function (which shouldn't be imported)